### ISSUE TITLE
Adding -c, --config options to read from TOML file

### ISSUE THREAD

**Issue by omalk98** [Yousef-Majidi/Learn2Blog#16]
Hi, I would like to add a feature to be able to read from a config file for you application.
The idea is to supply a -c or --config option and pass it a file path to be able to read from that config path instead of looking through your command history or memorizing the commands that you use regularly. 

This will most likely require changes to the CommandLineParser.cs file. 
This will also require the addition of a TOML parsing library, I was thinking of using [Tommy](https://www.nuget.org/packages/Tommy).
I would also like to make changes to how the command line reads arguments, the current way of reading command line options is non standard and will read it from the last parameter rather than the one directly after a flag. I would like to make it so that the input file always comes last.

**Comment by Yousef-Majidi** [comment]
Sounds good. Go ahead with adding `-c` support.

But could you explain a little bit further about the changes you would like to make to how the command line reads arguments? Could you provide an example and showcase how you're going to modify it? If it's an extensive change, I would ask you to create a separate issue for it and provide the details there, and I will take care of it at a later date. 

Thank you for your contribution.

**Comment by omalk98** [comment]
The current command line parser reads values as such:
```bash
# flag command_input flag_input
./Learn2Blog.exe -o inputSample.txt outputDirectory
```
I would like to change to behave in a more standard way, example:
```bash
# flag  flag_input command_input
./Learn2Blog.exe -o outputDirectory inputSample.txt
```
When adding more flags such as -c, it would become difficult to keep track of which flags map to which argument. keeping the input next to the flag will make it easier to use and understand.

**Comment by Yousef-Majidi** [comment]
I see. Thank you for catching this. 

Go ahead. But I think it would make more sense to always expect the `inputPath` as the very first argument before any flags: 

``` bash
./Learn2Blog.exe [inputPath] [flag] [flagPath]
```

But I plan to refactor my code next week. For now, you can go ahead with the change that you proposed, and I will implement what I described above in the next iteration.

**Comment by Yousef-Majidi** [comment]
implemented in #17
